src/hotspot/cpu/aarch64/aarch64_sve_ad.m4

```diff
@@ -2150,27 +2148,24 @@ VTEST_PARTIAL(anytrue, ne,      -1, NE)
 
 // ------------------------------ Vector insert ---------------------------------
 define(`VECTOR_INSERT_SMALL', `
-instruct insert$1_small`'(vReg dst, vReg src, $2 val, immI idx, vReg tmp, pRegGov pTmp, rFlagsReg cr)
+instruct insert$1_small`'(vReg dst, vReg src, $2 val, immI idx, pRegGov pTmp, rFlagsReg cr)
 %{
   predicate(UseSVE > 0 && n->as_Vector()->length() <= 32 &&
             n->bottom_type()->is_vect()->element_basic_type() == T_`'TYPE2DATATYPE($1));
   match(Set dst (VectorInsert (Binary src val) idx));
-  effect(TEMP tmp, TEMP pTmp, KILL cr);
+  effect(TEMP_DEF dst, TEMP pTmp, KILL cr);
   ins_cost(4 * SVE_COST);
-  format %{ "sve_index $tmp, $3, -16, 1\n\t"
-            "sve_cmpeq $pTmp, $tmp, ($idx-#16) // shift from [0, 31] to [-16, 15]\n\t"
+  format %{ "sve_index $dst, $3, -16, 1\n\t"
+            "sve_cmpeq $pTmp, $dst, ($idx-#16) // shift from [0, 31] to [-16, 15]\n\t"
             "sve_orr $dst, $src, $src\n\t"
             "sve_cpy $dst, $pTmp, $val\n\t# insert into vector ($1)" %}
```

Comment:
\n\t -> \t for the last line of the block.

Reply:
Thank you for your review. I have fixed this format problem.